deploy/helm/values.yaml

```diff
@@ -40,6 +40,8 @@ operator:
   # kubernetesBenchmarkEnabled the flag to enable CIS Kubernetes Benchmark scanner
   kubernetesBenchmarkEnabled: true
   # batchDeleteLimit the maximum number of config audit reports deleted by the operator when the plugin's config has changed.
+  vulnerabilityScannerScanOnlyCurrentRevisions: false
+  # vulnerabilityScannerScanOnlyCurrentRevisions the flag to only create vulnerability scans on the current revision of a deployment.
   batchDeleteLimit: 10
```

Comment:
```suggestion
  # batchDeleteLimit the maximum number of config audit reports deleted by the operator when the plugin's config has changed.
  batchDeleteLimit: 10
  # vulnerabilityScannerScanOnlyCurrentRevisions the flag to only create vulnerability scans on the current revision of a deployment.
  vulnerabilityScannerScanOnlyCurrentRevisions: false
```

Reply:
Haha it took me like 5 minutes to see what the difference was in your commit. Nice catch and thanks :)